arviz/data/inference_data.py

```diff
@@ -589,6 +593,103 @@ def to_dataframe(
             (dfs,) = dfs.values()
         return dfs
 
+    def to_zarr(self, store=None):
+        """Convert InferenceData to a :class:`zarr.hierarchy.group`.
```

Comment:
Once the docs are generated by Azure, please double check that this works. I think that outside the parameter type specification, intersphinx must be used to get links to work. We have some guidance on this in the [developer guide](https://arviz-devs.github.io/arviz/contributing/developer_guide.html#documentation)

Reply:
Should be working, since it is working on local documentation (`make html`).